docs/src/main/sphinx/connector/hive.rst

```diff
@@ -978,5 +978,6 @@ Hive 3 related limitations
       )
       AS <query>
 
-  Trino does not support gathering table statistics for Hive transactional tables.
-  You need to use Hive to gather table statistics with ``ANALYZE TABLE COMPUTE STATISTICS`` after table creation.
+* For transactional tables, Trino does not support gathering table statistics while processing
+  ``INSERT`` and ``CREATE TABLE AS SELECT`` requests, as well as ``UPDATE`` requests.  However,
```

Comment:
CREATE TABLE AS SELECT -> CREATE TABLE AS

see https://trino.io/docs/current/sql/create-table-as.html

consider you can `CREATE TABLE x AS TABLE other_table`
in such a case, there is no "SELECT"

Reply:
Thanks for the tip; removed SELECT.